GeoQuiz/app/src/main/java/com/bignerdranch/android/geoquiz/Question.kt

```diff
@@ -0,0 +1,5 @@
+package com.bignerdranch.android.geoquiz
+
+import androidx.annotation.StringRes
+
+data class Question(@StringRes val textResId: Int, val answer: Boolean)
```

Comment:
when having multiple params is useful to write them on separate lines for better readability https://developer.android.com/kotlin/style-guide#functions